src/main/java/bio/terra/cli/command/resource/create/GcpNotebook.java

```diff
@@ -130,7 +143,10 @@ protected void execute() {
             .machineType(machineType)
             .postStartupScript(postStartupScript)
             .metadata(Optional.ofNullable(metadata).orElse(Collections.emptyMap()));
-
+    if (instanceId == null){
+        createParams
+            .instanceId(generateInstanceId());
```

Comment:
currently the backend generates the ID.  Any reason not to do it there?  Then it would match the format generated for the UI.

Reply:
Currently, the documentation indicates that the generated instance ID will be a combination of the user's email and the time. That isn't the case presently. Instead, the generated instance ID appears to be the name of the resource. This is less useful, for debugging purposes.

Reply:
I don't object to the format change (though others may have opinions).  I just think the change should be in WSM, not the CLI.

Reply:
okay, thank you. I will refactor my change to create the instance ID in the WSM instead.

Reply:
I think you misunderstood.  I meant in the WSM code base, not this CLI repo.

Reply:
Okay, I've reverted the changes and instead revised the documentation to reflect the current behavior.